core/Azure.Mcp.Core/src/Areas/Tools/Commands/ToolsListCommand.cs

```diff
@@ -128,14 +196,22 @@ private static CommandInfo CreateCommand(string tokenizedName, IBaseCommand comm
                 required: arg.Required))
             .ToList();
 
+        var fullCommand = tokenizedName.Replace(CommandFactory.Separator, ' ');
+        // Strip the "azmcp " prefix from the command
+        var commandWithoutPrefix = fullCommand.StartsWith("azmcp ", StringComparison.OrdinalIgnoreCase)
+            ? fullCommand.Substring(6) // Remove "azmcp "
```

Comment:
We can have other servers (e.g. Fabric) using this class as it is a part of Core. I think it would be more robust to search for the first space/separator character and remove everything before that, for example.

Reply:
This PR is intended to add the functionality to list tool names only. I would suggest to clean up the code in another PR. I am trying to get this functionality in, so that partner teams who are adding new tools can use it.